src/Goerli-DssSpellCollateral.sol

```diff
@@ -30,58 +68,180 @@ contract DssSpellCollateralAction {
+            "RWA007-A: Monetalis",
+            GemAbstract(RWA007).symbol()
+        );
+    }
 
     function onboardNewCollaterals() internal {
-        // ----------------------------- Collateral onboarding -----------------------------
-        //  Add ______________ as a new Vault Type
-        //  Poll Link:
-
-        // DssExecLib.addNewCollateral(
-        //     CollateralOpts({
-        //         ilk:                   ,
-        //         gem:                   ,
-        //         join:                  ,
-        //         clip:                  ,
-        //         calc:                  ,
-        //         pip:                   ,
-        //         isLiquidatable:        ,
-        //         isOSM:                 ,
-        //         whitelistOSM:          ,
-        //         ilkDebtCeiling:        ,
-        //         minVaultAmount:        ,
-        //         maxLiquidationAmount:  ,
-        //         liquidationPenalty:    ,
-        //         ilkStabilityFee:       ,
-        //         startingPriceFactor:   ,
-        //         breakerTolerance:      ,
-        //         auctionDuration:       ,
-        //         permittedDrop:         ,
-        //         liquidationRatio:      ,
-        //         kprFlatReward:         ,
-        //         kprPctReward:
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     ILK,
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
-        // ChainLog Updates
-        // Add the new flip and join to the Chainlog
-        // address constant CHAINLOG        = DssExecLib.LOG();
-        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
-        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
-        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+        ChainlogAbstract CHANGELOG       = ChainlogAbstract(DssExecLib.LOG);
+        IlkRegistryAbstract REGISTRY     = IlkRegistryAbstract(DssExecLib.reg());
+        address MIP21_LIQUIDATION_ORACLE = CHANGELOG.getAddress("MIP21_LIQUIDATION_ORACLE");
+        address MCD_PSM_USDC_A           = CHANGELOG.getAddress("MCD_PSM_USDC_A");
+        address MCD_VAT                  = DssExecLib.vat();
+        address MCD_JUG                  = DssExecLib.jug();
+        address MCD_SPOT                 = DssExecLib.spotter();
+        address MCD_JOIN_DAI             = DssExecLib.daiJoin();
+
+        // --------------------------- RWA Collateral onboarding ---------------------------
+
+        // Onboard Monetalis: https://vote.makerdao.com/polling/TODO
+        onboardRwa007(CHANGELOG, REGISTRY, MIP21_LIQUIDATION_ORACLE, MCD_VAT, MCD_JUG, MCD_SPOT, MCD_JOIN_DAI, MCD_PSM_USDC_A);
     }
 
     function offboardCollaterals() internal {}
```

Comment:
I would remove this as not used in current spell

Reply:
Done